app/controllers/katello/api/v2/flatpak_remote_repositories_controller.rb

```diff
@@ -0,0 +1,74 @@
+module Katello
+  class Api::V2::FlatpakRemoteRepositoriesController < Katello::Api::V2::ApiController
+    include Katello::Concerns::FilteredAutoCompleteSearch
+
+    before_action :find_authorized_katello_resource, :except => [:index, :auto_complete_search, :mirror]
```

Comment:
When I try mirroring, `@flatpak_remote_repository` is still nil. Is it because `:mirror` is in the exception list for find_authorized_katello_resource?

Reply:
Aah I see..I am trying to not add any remote_repository permissions and base mirroring on product permissions..
Will need to update the mirror API to look for the remote repo outside of find_authorized_katello_resource.